lib/src/model/robotoff_nutrient_extraction.dart

```diff
@@ -0,0 +1,201 @@
+import 'package:json_annotation/json_annotation.dart';
+import '../interface/json_object.dart';
+
+part 'robotoff_nutrient_extraction.g.dart';
+
+@JsonSerializable()
+class RobotoffNutrientExtractionResult extends JsonObject {
+  final String? status;
+  final int? count;
+  final List<NutrientExtractionInsight>? insights;
+
+  const RobotoffNutrientExtractionResult({
+    this.status,
+    this.count,
+    this.insights,
+  });
+
+  NutrientExtractionInsight? get getLatestInsights {
+    insights?.sort((a, b) => a.completedAt!.compareTo(b.completedAt!));
```

Comment:
Please deal with the case where `completedAt` is null, as you don't reject that possibility.